package.json

```diff
@@ -1,6 +1,21 @@
 {
   "name": "saucelabs-connector",
   "version": "0.0.1",
+  "description": "Helps to connect local machine to the SauceLabs and start remote selenium webdriver on the side",
```

Comment:
`...and start a browser with the specified page url`
@VasilyStrelyaev please correct this